packages/uhk-common/src/models/uhk-products.ts

```diff
@@ -45,3 +48,77 @@ export const UHK_DEVICES: Array<UhkDeviceProduct> = [
     UHK_60_DEVICE,
     UHK_60_V2_DEVICE
 ];
+
+export interface UhkModule {
+    type: UhkProductTypes.Module;
+    id: number;
+    name: string;
+    slotId: ModuleSlotToId;
+    slotName: string;
+    i2cAddress: ModuleSlotToI2cAddress;
+    firmwareUpgradeSupported: boolean;
+    bootloaderPingReconnectMsg: string;
+}
+
+export const LEFT_HALF_MODULE: UhkModule = {
+    type: UhkProductTypes.Module,
+    id: 1,
+    name: 'Left Keyboard Half',
+    slotId: ModuleSlotToId.leftHalf,
+    slotName: 'LeftKeyboardHalf',
+    i2cAddress: ModuleSlotToI2cAddress.leftHalf,
+    firmwareUpgradeSupported: true,
+    bootloaderPingReconnectMsg: 'Cannot ping the bootloader. Please remove the bridge cable, and keep reconnecting the left keyboard half until you see this message.'
+};
+
+export const LEFT_KEY_CLUSTER_MODULE: UhkModule = {
+    type: UhkProductTypes.Module,
+    id: 2,
+    name: 'Left Key Cluster',
```

Comment:
The modules are one-sided, so please don't write left or right. Only capitalize the first word. So this is: `Key cluster`